cql3/statements/alter_keyspace_statement.cc

```diff
@@ -83,22 +90,67 @@ void cql3::statements::alter_keyspace_statement::validate(query_processor& qp, c
 
 future<std::tuple<::shared_ptr<cql_transport::event::schema_change>, std::vector<mutation>, cql3::cql_warnings_vec>>
 cql3::statements::alter_keyspace_statement::prepare_schema_mutations(query_processor& qp, const query_options&, api::timestamp_type ts) const {
+    using namespace cql_transport;
     try {
-        auto old_ksm = qp.db().find_keyspace(_name).metadata();
+        event::schema_change::target_type target_type = event::schema_change::target_type::KEYSPACE;
+        auto ks = qp.db().find_keyspace(_name);
+        auto ks_md = ks.metadata();
+
         const auto& tm = *qp.proxy().get_token_metadata_ptr();
         const auto& feat = qp.proxy().features();
+        auto ks_md_update = _attrs->as_ks_metadata_update(ks_md, tm, feat);
+        std::vector<mutation> muts;
+        std::vector<sstring> warnings;
+
+        if (ks.get_replication_strategy().uses_tablets() /* && !rf_map.empty() */) {
+            qp.db().real_database().validate_keyspace_update(*ks_md_update);
+            if (auto repl_strategy = _attrs->get_replication_strategy_class();
```

Comment:
This check looks excessive. It effective validates that we're not changing the replication strategy from N.T.S. to something else, but the same check exists in `alter_keyspace_statement::validate()` already

Reply:
OK, removed.